asapdiscovery-docking/asapdiscovery/docking/scripts/make_docked_complexes.py

```diff
@@ -3,19 +3,17 @@
 """
 import argparse
 import os
-import sys
 
 import pandas
-from openeye import oechem
-
-sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 from asapdiscovery.data.openeye import load_openeye_pdb  # noqa: E402
 from asapdiscovery.data.openeye import load_openeye_sdf  # noqa: 402
 from asapdiscovery.data.openeye import save_openeye_pdb  # noqa: 402
 from asapdiscovery.data.openeye import split_openeye_mol  # noqa: E402
+from asapdiscovery.data.openeye import combine_protein_ligand
+from openeye import oechem
```

Comment:
now that #161 is merged you should import `oechem` from `asapdiscovery.data.openeye`

Reply:
should be fixed in [eb14cdc](https://github.com/choderalab/asapdiscovery/pull/160/commits/eb14cdcac7667f385ed81ef100e07b4a0735015f)